client-v2/package.json

```diff
@@ -14,7 +14,7 @@
     "dev-server": "yarn compileWithBabel $(yarn bin)/webpack-dev-server --config config/webpack.config.dev.js",
     "dev": "yarn install && yarn dev-server --open",
     "test": "yarn jest",
-    "test-integration": "BUILD_ENV=integration yarn build && node ./integration/run.js",
+    "test-integration": "BUILD_ENV=integration echo 'RUNNING ON EXISTING PROD BUILD - run yarn build before this to catch changes since last build' && node ./integration/run.js",
```

Comment:
Might be worth two commands to differentiate between dev (where we probably want the convenience of build + run) and prod (where we just want run)?

Reply:
I thought about this but just didn't know what to call it - I will append `ci` to the one that doesn't built.